components/organisms/o-personal-details.vue

```diff
@@ -123,214 +121,201 @@
                 <span class="sf-checkbox__label no-flex">
                   {{ $t("I accept ") }}
                 </span>
-                <SfButton class="sf-button sf-button--text terms" @click.prevent="openTermsAndConditionsModal">
+                <SfButton
+                  class="sf-button sf-button--text terms"
+                  @click.prevent="openTermsAndConditionsModal"
+                >
                   {{ $t("Terms and conditions") }}
                 </SfButton>
               </template>
             </SfCheckbox>
           </div>
         </template>
       </template>
-      <div class="form__action">
-        <SfButton
-          class="sf-button--full-width form__action-button"
-          :disabled="createAccount ? $v.$invalid : $v.personalDetails.$invalid"
-          @click="sendDataToCheckout"
-        >
-          {{
-            $t(isVirtualCart ? "Continue to payment" : "Continue to shipping")
-          }}
-        </SfButton>
-        <SfButton
-          v-if="!currentUser"
-          class="sf-button--full-width sf-button--text form__action-button form__action-button--secondary mobile-only"
-          @click="login"
-        >
-          {{ $t("or login to your account") }}
-        </SfButton>
-      </div>
+
+      <SfButton
+        class="action-button"
+        :disabled="createAccount ? $v.$invalid : $v.personalDetails.$invalid"
+        @click="sendDataToCheckout"
+      >
+        {{ $t(isVirtualCart ? "Continue to payment" : "Continue to shipping") }}
+      </SfButton>
     </div>
   </div>
 </template>
 <script>
-import { required, minLength, email, sameAs } from 'vuelidate/lib/validators';
-import { PersonalDetails } from '@vue-storefront/core/modules/checkout/components/PersonalDetails';
-import { SfInput, SfButton, SfHeading, SfCheckbox, SfCharacteristic } from '@storefront-ui/vue';
-import { ModalList } from 'theme/store/ui/modals'
-import { mapActions } from 'vuex';
+import { required, minLength, email, sameAs } from "vuelidate/lib/validators";
```

Comment:
These are the formatting changes, Please do not mix the formatting changes with functional change.s

Reply:
Ok will fix this